marimbabot_planning/launch/marimbabot.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+    <node name="lilypond_to_hitpoint" pkg="marimbabot_planning" type="lilypond_to_hitpoint.cpp"/>
```

Comment:
node name defined as "move_lilypond_to_hitpoints" in the lilypond_to_hitpoint.cpp. Would be nice to keep the node names as same.

Reply:
I think lilypond_to_hitpoint.cpp isn't correct ether

Reply:
I debated too but here they included .py extension for the node.
http://wiki.ros.org/roslaunch/XML

Reply:
This is the case because the py file is executed as is. The cpp file gets compiled and the resulting binary is named `lilypond_to_hitpoint` most of the time.